backend/api-service/src/backend.go

```diff
@@ -42,6 +42,7 @@ func (i *Inventory) applyTransaction(txn *InventoryTransaction) error {
 type DatabaseBackend interface {
 	DeleteItem(ctx context.Context, id string) error
 	DeleteLocation(ctx context.Context, id string) error
+	DeleteAlert(ctx context.Context, id string) error
```

Comment:
Nit: could these be kept in alphabetical order?

Reply:
Yep - done.